app/src/main/AndroidManifest.xml

```diff
@@ -119,5 +119,7 @@
     <activity android:name=".testing.ReviewCardFragmentTestActivity"
       android:screenOrientation="portrait"
       android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity android:name=".walkthrough.WalkthroughActivity"
+      android:screenOrientation="portrait" />
```

Comment:
Just press
`Control + Alt + L` to reformat everything. 

(This is not important but we are doing this just to make an extra commit and see if Circle CI passes.)